js/components/utils.js

```diff
@@ -314,9 +361,7 @@ var Utils = {
   highlight: function(target, opacity, color) {
     target.style.opacity = opacity;
     target.style.color = color;
-    target.style.WebkitFilter = "drop-shadow(0px 0px 3px rgba(255,255,255,0.8))";
-    target.style.filter = "drop-shadow(0px 0px 3px rgba(255,255,255,0.8))";
-    target.style.msFilter = "progid:DXImageTransform.Microsoft.Dropshadow(OffX=0, OffY=0, Color='#fff')";
+    target.style.textShadow = "0px 0px 3px rgba(255, 255, 255, 0.5), 0px 0px 6px rgba(255, 255, 255, 0.5), 0px 0px 9px rgba(255, 255, 255, 0.5)";
```

Comment:
Yes, ideally this should match the mixin. There is no way to link them as far as I know, so I will add comments to both files